src/components/ClassList.tsx

```diff
@@ -17,6 +17,15 @@ import type { ClassCardInfo } from "models/Class";
 export const ClassList: React.FC<{ classInfo: ClassCardInfo[] }> = ({
     classInfo,
 }) => {
+    function convertToAmPm(timeInMinutes: Int) {
```

Comment:
did you mean to put `number` as the type?

Reply:
Let's also add some documentation comments on helper functions to maintain a standard!

Reply:
LMAO vscode so laggy it didnt catch that